src/components/icon-button/icon-button.pw.tsx

```diff
@@ -0,0 +1,215 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import IconButtonComponent from "./component.test-pw";
+import { iconButton } from "../../../playwright/components/index";
```

Comment:
**suggestion**: if you've changed the index/locator name from `iconButton` to `button` you can then do this for the import `import { button as iconButton } from "../../../playwright/components/index"`. This way you can keep the name of the locator in your tests and ensure they're still clear what's being tested etc.